workspace.go

```diff
@@ -349,6 +350,10 @@ type WorkspaceCreateOptions struct {
 	// A list of tags to attach to the workspace. If the tag does not already
 	// exist, it is created and added to the workspace.
 	Tags []*Tag `jsonapi:"relation,tags,omitempty"`
+
+	// Optional: Whether to enabled drift detection for the workspace.
+	// Requires remote execution mode, TFCB entitlement, and a valid agent pool to work
+	DriftDetection *bool `jsonapi:"attr,drift-detection,omitempty"`
```

Comment:
It looks like the attributes are listed in alphabetical order (after the type). Could you move this new attribute to maintain that ordering?